csrankings-a.csv

```diff
@@ -1487,7 +1488,6 @@ Andrew Sohn,NJIT,http://cs.njit.edu/people/sohn.php,NOSCHOLARPAGE
 Andrew T. Campbell,Dartmouth College,http://www.cs.dartmouth.edu/~campbell,rthco5oAAAAJ
 Andrew T. Duchowski,Clemson University,http://andrewd.ces.clemson.edu,J_oQ4-oAAAAJ
 Andrew Taylor,UNSW,http://www.cse.unsw.edu.au/~andrewt,bODP_fkAAAAJ
-Andrew Tolmach,Portland State University,http://web.cecs.pdx.edu/~black,NOSCHOLARPAGE

```

Comment:
This needs to be restored (with the correct home page URL :) ) - all aliases need to be present.

Reply:
Thanks. Fixed.